.travis.yml

```diff
@@ -44,12 +36,10 @@ install:
     # - pandas will bring dateutil numpy pytz setuptools six and scipy, we
     #   might want to avoid the later by installing all dependencies manually
     #   except scipy and install pandas with --no-deps
-    # - fixed versions of numpy and pytables because of incompatibilities
-    #   between pytables <= 3.4 and numpy >= 1.16
     # - pandas >= 0.20.0 is required since commit 01669f2024a7bffe47cceec0a0fd845f71b6f7cc
     #   (issue 702 : fixed bug when writing metadata using HDF format)
     - conda create -n travisci --yes python=${TRAVIS_PYTHON_VERSION:0:3}
-            "numpy==1.15.2" "pandas>=0.20" "pytables==3.4.4" matplotlib xlrd openpyxl
+            "numpy=>1.13" "pandas>=0.20" "pytables=>3.5" matplotlib xlrd openpyxl
```

Comment:
you should **not** specify pytables >=3.5 as we do not really need it (besides isn't => a typo instead of >=?)